ui/src/common/components/CitationsByYearGraph/CitationsByYearGraph.jsx

```diff
@@ -129,11 +129,41 @@ class CitationsByYearGraph extends Component {
     );
   }
 
+  renderCitationsGraph() {
+    const { seriesData } = this.state;
+    const currentYear = new Date().getFullYear();
+
+    if (seriesData?.slice(-1)[0]?.x === currentYear) {
+      return [
+          <LineSeries
+            sizeType="literal"
+            onNearestX={this.onGraphMouseOver}
+            data={seriesData.slice(0, seriesData.length - 1)}
```

Comment:
can you assign the data you're passing to some meaningful vars before passing them to the `LineSeries`? Cause it's a bit hard to understand if you haven't seen a graph before